Test/Unit/Model/Api/ObjectTest.php

```diff
@@ -98,7 +98,7 @@ public function reload()
 class MockOmiseObject implements \ArrayAccess
 {
     // Store the attributes of the object.
-    protected $_values = array();
+    protected $_values = [];
```

Comment:
Protected var name starting with `_`

Reply:
@jonrandy Thanks
Code-refactoring PR will be coming in soon!

Reply:
@guzzilar no necessary to change in pr only few places.. change all in entire plugin..

Reply:
@jacekstan yup, so changing entire plugin will be coming in an another pull request once `tesco-lotus` feature is merged (or maybe before) (I updated array syntax & opened this pull request just to continue from a conversation at the previous PR #123)

Reply:
so why this PR *now* is necessary?
and why there is problem to follow all plugin with those changes?